server/src/main/java/org/candlepin/policy/criteria/CriteriaRules.java

```diff
@@ -65,7 +65,7 @@ public CriteriaRules(Configuration config, ConsumerCurator consumerCurator) {
         // consumer UUID
         Consumer hostConsumer = null;
         if (consumer.getFact("virt.uuid") != null) {
-            hostConsumer = consumerCurator.getHost(consumer.getFact("virt.uuid"), consumer.getOwner());
+            hostConsumer = consumerCurator.getHost(consumer);
```

Comment:
iirc, @mstead  or @awood  had investigated and found that the time difference between when the owner is specified and when not is negligible.
was this investigated against only one of  mysql/postgres or both?